Desktop/Application/MaxMix/Services/Communication/CommunicationService.cs

```diff
@@ -200,6 +203,16 @@ private void HandleReconnection()
                         }
                     }
                 }
+                else
+                {
+                    // Send a regular heartbeat
+                    if (_heartbeatWatch.Elapsed.TotalMilliseconds > 5000)
```

Comment:
Extract this to a constant?

Reply:
Should we reduce the hearbeat time? 5s seems a bit excesive, doesn't it?

Reply:
set it to 2sec